packages/wallets/src/features/cashier/modules/WithdrawalCrypto/components/WithdrawalCryptoReceipt/WithdrawalCryptoReceipt.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { WalletButton, WalletCard, WalletText } from '../../../../../../components';
+import ArrowDown from '../../../../../../public/images/ic-back-arrow.svg';
+import { WithdrawalCryptoDestinationAddress } from './components';
+import './WithdrawalCryptoReceipt.scss';
+
+type TProps = {
+    onClose: () => void;
+    withdrawalReceipt: {
```

Comment:
we already have this type [here](https://github.com/binary-com/deriv-app/pull/11577/files#:~:text=type%20TWithdrawalReceipt%20%3D,%7D%3B)
maybe we can create a local types.ts file inside this module that shared reusable types?